javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/JaversSchemaManager.java

```diff
@@ -73,6 +77,14 @@ private void alterCommitIdColumnIfNeeded() {
         }
     }
 
+    /**
+     * This method is needed for upgrading TEXT columns to VARCHAR(MAX) since TEXT is deprecated.
+     */
+    private void alterMssqlTextColumns() {
```

Comment:
one more question, did you tested also this schema migration script? I mean did you run new version of javers with old schema?